tests/Facebook/InstantArticles/Elements/Validators/InstantArticleValidatorTest.php

```diff
@@ -113,6 +113,12 @@ public function testInstantArticle(): void
         $this->assertEquals($expected, $result);
 
         $warnings = InstantArticleValidator::check($article);
+
+        foreach($warnings as $warning) {
+          var_dump($warning->__toString());
+        }
```

Comment:
I guess you can remove this logging.

Reply:
I'm sorry, this looks to be old code.

Reply:
No problem, dropping it!
Thanks for flagging!